src/pages/docs/oauth/index.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: OAuth
+description: This describes the oauth config used for applications
+tags:
+  - ShipEngine Connect
+  - OAuth
+---
+
+# Mono Auth Overview
+The ShipEngine Connect platform uses an internal system called `MonoAuth` to handle Oauth 1 and 2 auth handshake flows.
+This internal system works with a DSL that you define inside of your application definition, and supports property injection in the form of tags.
+
+## Installation Process
+- Define your application as using OAuth as seen in the following [example]()
+- Publish your application to ShipEngine Connect using the `connect publish`
+- Contact a member of the [ShipEngine Connect Team]() in order to establish a MonoAuth installation
+    - Provide a ClientId & ClientSecret used for initiating the oauth workflow
+- Once the installation has been created you will recieve back a callback url Ex: `https://authbridge-sec.auctane.com/authflow/{application_code}/accept`
```

Comment:
does the user know how to obtain their `{application_code}` ?

Reply:
Yes, I was letting them know that they will be given back a url shaped like that.

Reply:
I don't think they even need to know about this URL.